src/Screens/HomeTabs/Shows/Shows.tsx

```diff
@@ -1,13 +1,38 @@
 import React from "react"
-import { Text } from "palette"
-import { Tabs } from "react-native-collapsible-tab-view"
+import { GlobalStore } from "store/GlobalStore"
+import { graphql, useLazyLoadQuery } from "react-relay"
+import { extractNodes } from "helpers/utils/extractNodes"
+import { ShowsTabQuery } from "__generated__/ShowsTabQuery.graphql"
+import { ShowsTabFlatList } from "./ShowsTabFlatList/ShowsTabFlatList"
 
 export const Shows = () => {
-  return (
-    <Tabs.ScrollView>
-      <Text>Shows</Text>
-      <Text>Shows</Text>
-      <Text>Shows</Text>
-    </Tabs.ScrollView>
+  const partnerID = GlobalStore.useAppState((state) => state.activePartnerID)
+  const data = useLazyLoadQuery<ShowsTabQuery>(
+    graphql`
+      query ShowsTabQuery($partnerID: String!) {
+        partner(id: $partnerID) {
+          showsConnection(first: 2) {
+            totalCount
+            edges {
+              node {
+                name
+                startAt
+                endAt
+                formattedStartAt: startAt(format: "MMMM D")
+                formattedEndAt: endAt(format: "MMMM D, YYYY")
+                coverImage {
+                  url
+                }
+              }
+            }
+          }
+        }
+      }
+    `, //@ts-ignore
+    { partnerID }
```

Comment:
Lets avoid adding @ts-ignore

```suggestion
    { partnerID: partnerID! }
```